content/en/docs/devops-user-guide/how-to-use/set-ci-node.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Set CI Node for Dependency Cache"
+keywords: 'kubernetes, docker, kubesphere, jenkins, cicd, pipeline'
+description: 'How to Set CI Node for dependency cache of KubeSphere pipeline '
+---
+
+## Introduction
+
+Generally, applications often need to pull a lot of dependencies during the build process. It might cause some issues like long pulling time, or unstable network causing failure. In order to make build robust, and to speed up the build by using cache, we recommend you configure one or a set of CI nodes which the system schedules the task of CI/CD pipelines or S2I/B2I builds running on.
+
+## Label CI Nodes
+
+1. Log in KubeSphere with `admin` account, navigate to **Platform → Infrastructure**.
```

Comment:
This step should be updated and synchronized to v3.0.0, see the preview page: https://deploy-preview-367--kubesphere-v3.netlify.app/docs/devops-user-guide/how-to-use/set-ci-node/#label-ci-nodes

Reply:
fixed